man/docker-create.1.md

```diff
@@ -267,6 +268,10 @@ This value should always larger than **-m**, so you should always use this with
 **-t**, **--tty**=*true*|*false*
    Allocate a pseudo-TTY. The default is *false*.
 
+**--tmpfs**=[] Create a tmpfs mount
+
+   Mount a temporary filesystem (tmpfs) mount within a container. For example,  `--tmpfs /tmp:rw,size=787448k,mode=1777` mounts a tmpfs at /tmp within the container. The mount copies the underlying content into the tmpfs). The supported options are the same as the Linux default mount flags. If you do not specify any options, we default to the following options: `rw,noexec,nosuid,nodev,size=65536k`.
```

Comment:
I added an explanation.  I could add an example of 

docker -d --tmpfs /etc --tmpfs /run httpd